cf-agent/verify_files.c

```diff
@@ -760,10 +754,33 @@ static bool SaveBufferCallback(const char *dest_filename, void *param, NewLineMo
     return true;
 }
 
-static PromiseResult RenderTemplateMustache(EvalContext *ctx, const Promise *pp, const Attributes *attr,
-                                            EditContext *edcontext, const char *template)
+inline static bool CreateFalseWasSpecified(const Promise *pp)
```

Comment:
We do `static inline` in other places.

Reply:
Actually, this should probably not be `inline` as it's rarely called

Reply:
I think it makes sense to have it as `inline`.